src/app_config.cpp

```diff
@@ -232,6 +234,23 @@ ConfigOpt *opts[] =
 ConfigJson user_config(opts, sizeof(opts) / sizeof(opts[0]), EEPROM_SIZE, CONFIG_OFFSET);
 ConfigJson factory_config(opts, sizeof(opts) / sizeof(opts[0]), EEPROM_SIZE, FACTORY_OFFSET);
 
+uint32_t
+getConfigVersion() {
```

Comment:
Should use the same snake case style as the other functions
```suggestion
config_version() {
```